signed-memo/program/src/processor.rs

```diff
@@ -0,0 +1,107 @@
+//! Program state processor
+
+use solana_program::{
+    account_info::AccountInfo, entrypoint::ProgramResult, msg, program_error::ProgramError,
+    pubkey::Pubkey,
+};
+use std::str::from_utf8;
+
+/// Instruction processor
+pub fn process_instruction(
+    _program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    input: &[u8],
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+    let mut missing_required_signature = false;
+    for account_info in account_info_iter {
+        if let Some(address) = account_info.signer_key() {
+            msg!("Signed by {:?}", address);
+        } else {
+            msg!("Not signed! {:?}", account_info.key);
```

Comment:
I think I'd be ok with removing this log entirely.    Less chance of something writing a really dump log parser that just matches on address and considers that to be a success

Reply:
Should we suppress all logs unless all the signer-checks pass, then?